docs/integrations/catalyst.rst

```diff
@@ -23,7 +23,7 @@ Create the **NeptuneLogger**
 ----------------------------
 .. code-block:: python3
 
-    from catalyst.contrib.dl.callbacks.neptune import NeptuneLogger
+    from catalyst.contrib.callbacks import NeptuneLogger
```

Comment:
I see that Logger is in the neptune_logger.py file: https://github.com/catalyst-team/catalyst/blob/df9e07f962ca61986a31c602a4511ad983dad028/catalyst/contrib/callbacks/neptune_logger.py#L18

So probably: `catalyst.contrib.callbacks.neptune_logger import NeptuneLogger`

Reply:
```suggestion
    from catalyst.dl import NeptuneLogger
```